### DISCUSSION TITLE
Maandag 25-11-2024

### DISCUSSION THREAD

**Discussion by Annevd** [rutgerkock/buurtcampus-oost#12] [Stand-ups]
- Hoe gaat het met je en waar ben je nú mee bezig?
- Wat wil je deze week bereiken (aan werken)?
- Waar loop je tegen aan?

**Comment by Nazneen05x** [comment]
- Hoe gaat het met je en waar ben je nú mee bezig?
 I am okey. Vorige week ben ik begonnen met het opnieuw bouwen van het menu. 

- Wat wil je deze week bereiken (aan werken)?
Deze week wil ik beginnen met het maken van het dropdown menu.

- Waar loop je tegen aan?
Enige waar ik op dit moment tegenaan loop is het mobiele menu fullscreen maken. Verder ben ik goed op weg, het mobiele menu is gestijld en het  werkt.

**Comment by wingsvn** [comment]
**hoe gaat het met je en waar ben je nu mee bezig?**
het gaat goed, ik ben bezig geweest met de must haves voor de agenda pagina. Tot zover heb ik het contrast van de agenda pagina gechecked. Verder heb ik aanmeld knoppen toegevoegd voor de workshop kaartjes. 

**wat wil je deze week bereiken (aan werken)?**
ik wil werken aan de rest van de must haves van de agenda pagina, dus de testen. Ik wil ook nog kijken naar het aanmeld formulier voor de workshop kaartjes.

**waar loop je tegen aan?**
niet iets specifieks momenteel, meer met het werken aan meerdere dingen tegelijkertijd.

**Comment by Lmikkers** [comment]
### Hoe gaat het met je en waar ben je nu mee bezig?
### Wat wil je deze week bereiken (aan werken)?
### Waar loop je tegen aan?

**Comment by rutgerkock** [comment]
**Hoe gaat het met je en waar ben je nú mee bezig?**
Bezig met m'n component en begin aan nieuwe Harry.

**Wat wil je deze week bereiken (aan werken)?**
Component afmaken en in ieder geval design van nieuwe Harry af.

**Waar loop je tegen aan?**
Harry ingewikkeld.

**Comment by Annevd** [comment]
- **Hoe gaat het met je en waar ben je nú mee bezig?**
Ik ben bezig geweest met mijn formulier component, en heb hiervan feedback verwerkt van de sprint review van afgelopen donderdag. Verder werk ik verder aan het projectplan en ben ik bezig een responsive bug op te lossen op de contact pagina.
- **Wat wil je deze week bereiken (aan werken)?**
Ik hoop de responsive bug opgelost te hebben en daarmee verder te kunnen aan andere must haves en should haves.
- **Waar loop je tegen aan?**
Tegen de responsive bug op android. Ik ga aan een docent vragen hoe ik het op kan lossen.
